src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/writefinaldecision/WriteFinalDecisionPreviewDecisionService.java

```diff
@@ -209,7 +209,7 @@ private void setEntitlements(WriteFinalDecisionTemplateBodyBuilder builder, Sscs
 
     protected void setDescriptorsAndPoints(WriteFinalDecisionTemplateBodyBuilder builder, SscsCaseData caseData) {
         List<String> dailyLivingAnswers = ActivityType.DAILY_LIVING.getAnswersExtractor().apply(caseData);
-        if (dailyLivingAnswers != null) {
+        if (dailyLivingAnswers != null && !"notConsidered".equals(caseData.getPipWriteFinalDecisionDailyLivingQuestion())) {
```

Comment:
Can you use the AwardType.NOT_CONSIDERED.getKey() here instead of hardcoded string?

Reply:
Good catch - thanks - have now fixed